content/packit-as-a-service.md

```diff
@@ -70,14 +70,20 @@ An example of Packit Service pull request:
 
 ## FAQ
 
-### Can I use packit service as soon as I install it into my repository?
+### Can I use the packit service as soon as I install it into my repository?
 
-Thanks for your interest in Packit Service! To be able to start using service, we need to put you on a whitelist: feel free to get in touch with us or we will reach out to you once we add you to whitelist.
+Thanks for your interest in Packit Service! You need to be whitelisted in order
+to start using the service which is an action to be done by us. We are now
+on-boarding Fedora contributors (with a Fedora Account System account). Once we
+put you on the whitelist, we'll get in touch with you.
```

Comment:
playing with sentences ...

Thanks for your interest in Packit Service!
In order to start using the service you need to be whitelisted, which is an action to be done by us.
Once we put you on the whitelist, we'll get in touch with you.
We are now on-boarding Fedora contributors (with a Fedora Account System account).